Packs/CofenseTriage/pack_metadata.json

```
@@ -1,16 +1,16 @@
 {
-  "name": "Cofense Triage",
-  "description": "Use the Cofense Triage integration to manage reports and attachments.",
-  "support": "xsoar",
-  "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
-  "email": "",
-  "created": "2020-04-14T00:00:00Z",
-  "categories": [
-    "Data Enrichment & Threat Intelligence"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
-}
+    "name": "Cofense Triage",
+    "description": "Use the Cofense Triage integration to manage reports and attachments.",
+    "support": "xsoar",
+    "currentVersion": "1.1.0",
+    "author": "Cortex XSOAR",
```

You will be the owners of this Pack.
```suggestion
    "author": "Cofense",
```